Documentation/config/core.txt

```diff
@@ -591,8 +591,11 @@ core.multiPackIndex::
 	multi-pack-index design document].
 
 core.sparseCheckout::
-	Enable "sparse checkout" feature. See section "Sparse checkout" in
-	linkgit:git-read-tree[1] for more information.
+	Enable "sparse checkout" feature. If "false", then sparse-checkout
+	is disabled. If "true", then sparse-checkout is enabled with the full
+	.gitignore pattern set. If "cone", then sparse-checkout is enabled with
+	a restricted pattern set. See linkgit:git-sparse-checkout[1] for more
```

Comment:
nit: the term "... pattern set" threw me a bit here.  I guess I'm wondering if "set" is a noun or verb here.

Would it be better to say something about the pattern matching mode.  As in: If "true", then sparse-checkout is enabled using the rules for .gitignore-style pattern matching.  If "cone, then it is enabled using the rules for cone-style matching.

It seems like this config setting is just changing matching algorithm.
Whereas, saying "... pattern set" implies that we have 2 cached "sets" and are deciding which to install.